nexus/src/external_api/http_entrypoints.rs

```diff
@@ -5732,6 +5732,71 @@ impl NexusExternalApi for NexusExternalApiImpl {
             .await
     }
 
+    async fn system_update_get_target_release(
```

Comment:
nit: I believe that elsewhere, we've generally used `subject_verb` -- as in `blueprint_target_set`.  I might just call this `target_release_get` (or `target_release_view` -- we use "view" for blueprint targets) and `target_release_set`.

This goes for the datastore methods, too.

Reply:
Thanks, renamed in 5fe4bb31b. But I took those names from the repo depot methods, which should also maybe be updated.